app/src/test/java/org/example/app/PostgresTestContainer.java

```diff
@@ -0,0 +1,37 @@
+package org.example.app;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class PostgresTestContainer implements BeforeAllCallback, AfterAllCallback {
+    private PostgreSQLContainer<?> postgres;
+
+    @Override
+    public void beforeAll(ExtensionContext context) {
+        postgres = new PostgreSQLContainer<>("postgres:14.11")
+                .withDatabaseName("testdb")
+                .withUsername("postgres")
+                .withPassword("postgres")
+                .withExposedPorts(5432);
+
+        postgres.start();
+        System.setProperty("spring.flyway.url",

```

Comment:
You are right properties `spring.flyway.*` are not needed to run tests.
But let's leave it as it is to keep this configuration explicit.